yelp_nlp/rnn/load.py

```diff
@@ -0,0 +1,53 @@
+from dataclasses import dataclass, field
+import enum
+from attr import define
+from typing import List, Tuple
+import numpy as np
+import pandas as pd
+import jsonlines as jsonl
+
+import torch
+from torch.utils.data import Dataset
+from sklearn.model_selection import train_test_split
+from yelp_nlp.logging_utils import new_logger
+from yelp_nlp.rnn.config import LogLevels
+
+logger = new_logger(LogLevels.debug.value)
+
+
+def split_df(
+    df: pd.DataFrame, test_size: float = 0.25
+) -> Tuple[pd.DataFrame, pd.DataFrame]:
+
+    tr_idx, val_idx = train_test_split(df.index.values, test_size=test_size)
+    return df.iloc[tr_idx], df.iloc[val_idx]  # type: ignore
+
+
+@define
+class CorpusDataset(Dataset):
```

Comment:
maybe this belongs in models.py

Reply:
Feels better to decouple this from the model class, as potentially there could be multiple models in the file.